python/paddle/jit/dygraph_to_static/__init__.py

```diff
@@ -0,0 +1,27 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+from . import convert_operators
+
+from . import convert_call_func
+from .convert_call_func import *
+
+from . import variable_trans_func
+from .variable_trans_func import *
+
+__all__ = []
+__all__ += convert_call_func.__all__
+__all__ += variable_trans_func.__all__
```

Comment:
Same as above, does the code `__all__ += convert_operators.__all__` need to be added here?

Reply:
I think it is better just be same as that in the past. Even if we would like to change, it is better to be in another PR.

Reply:
OK. LGTM. Thanks for your reply.